pom.xml

```diff
@@ -72,7 +72,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <declared.ozone.version>${ozone.version}</declared.ozone.version>
 
     <!-- Apache Ratis version -->
-    <ratis.version>2.4.2-8b8bdda-SNAPSHOT</ratis.version>
+    <ratis.version>2.4.2-8b8bdda-20221130.230506-1</ratis.version>
```

Comment:
Where did we get this version string from?

Reply:
There were transitive dependencies on `2.4.2-8b8bdda-SNAPSHOT` and `2.4.2-8b8bdda-20221130.230506-1`. There was no help from [mavenCentral](https://central.sonatype.dev/artifact/org.apache.ratis/ratis-tools/2.4.1/overview), so looking at the date, I made a call to use the latter version.

Reply:
Understood. @nishitpatira use `2.4.2-8b8bdda-SNAPSHOT` that are for the same commit hash. The hex code post `2.4.2` is the commit hash. So both are pointing to the same version of the code.